packages/Molecules/Form/fields/fieldRplselect.vue

```diff
@@ -1,118 +1,384 @@
 <template>
-  <multiselect
-    :options="options"
-    :value="localValue"
-    v-bind="selectOptions"
-    :placeholder="schema.placeholder"
-    :custom-label="customLabel"
-    :max="schema.max || null"
-    :disabled="disabled"
-    @input="updateSelected"
-    @search-change="onSearchChange"
-    @tag="addTag"
-    :option-height="selectOptions.optionHeight"
-  >
-  </multiselect>
+  <div class="rpl-select" :class="{'rpl-select--open' : isOpen}">
+    <div v-if="$breakpoint.s" class="rpl-select__native">
+      <select :multiple="schema.multiselect" v-model="value">
+        <option v-if="!schema.multiselect" disabled value="">{{placeholder}}</option>
+        <option :value="option.id" v-for="(option) in options" :key="option.id">{{option.name}}</option>
+      </select>
+      <rpl-icon class="rpl-select__trigger-icon" symbol="down" color="primary" />
+    </div>
+    <div class="rpl-select__inner">
+      <div
+        class="rpl-select__trigger"
+        tabindex="0"
+        aria-haspopup="listbox"
+        :aria-labelledby="`${schema.model}-rpl-select-trigger ${schema.model}-rpl-select-value`"
+        :aria-expanded="isOpen ? 'true' : false"
+        role="button"
+        ref="trigger"
+        @click="toggleOpen"
+        v-on:keyup.enter.prevent="toggleOpen"
+      >
+        <template v-if="value && value.length > 0">
+          <span :id="`${schema.model}-rpl-select-value`">{{selectedTitles}}</span>
+        </template>
+        <span v-else :id="`${schema.model}-rpl-select-trigger`">{{placeholder}}</span>
+        <rpl-icon class="rpl-select__trigger-icon" symbol="down" color="primary" />
```

Comment:
While showing this icon for multi-select on mobile makes sense. It looks strange on desktop with a small window, having an arrow beside a multi-select `<select>` box.

<img width="357" alt="Screen Shot 2019-03-27 at 6 55 06 pm" src="https://user-images.githubusercontent.com/12739575/55059039-00def500-50c2-11e9-8457-e1293f333738.png">

Not sure what to do about that.

Reply:
agreed, the icon is weird for multiselect on mobile, have removed.